micro-benchmarks/DRB172-parallelfor-orig-no.c

```diff
@@ -0,0 +1,60 @@
+/**
+ * BT.C: This file is part of kernel of the NAS Parallel Benchmarks 3.0 bt suit.
+ * Tsan and Intel Inspector can not correctly analysis the nested parallel for, and report a false postive.
+*/
+
+#include <stdio.h>
+
+
+int main(int argh, char* argv[]){
+  int i,j,k,m;
+  int nthreads = 1;
+  double u[12][12][12][5];
+  double tmp1,tmp2,tmp3;
+  double fjac[12][12][12][5][5];
+
+  /*initial m */
+  #pragma omp for
+  for (i = 1; i < 12-1; i++) {
+    for (j = 1; j < 12-1; j++) {
+        for (k = 0; k < 12; k++){
+            for (m = 0; m < 5; m++){
+            u[i][j][k][m] = 2.5;
+      }
+    }
+    }
+  }
+    #pragma omp parallel
+    {
+  #pragma omp for
+  for (i = 1; i < 12-1; i++) {
```

Comment:
This looks very similar to the previous file. Why do you include both? What different patterns are you trying to show?

Reply:
Same as above. Do we need this kernel?